src/components/create-job-form/pipeline-form-promote.js

```diff
@@ -100,7 +99,11 @@ export const PipelineFormPromote = ({
                   {smallDeploymentName(dep.name)}{' '}
                   {dep.activeTo
                     ? `(${formatDateTime(dep.activeFrom)})`
-                    : `(currently active)`}
+                    : `(currently active)`}{' '}
+                  {dep.gitCommitHash
+                    ? `${dep.gitCommitHash.substring(0, 7)}`
+                    : ` `}{' '}
+                  {dep.gitTags ? `, ${dep.gitTags}` : ` `}{' '}
```

Comment:
minor optimization suggestion
```suggestion
                    : `(currently active)`}
                  {dep.gitCommitHash && ` ${dep.gitCommitHash.substring(0, 7)}`}
                  {dep.gitTags && `, ${dep.gitTags}`}
```